web/src/app/pages/signin/signin.controller.js

```diff
@@ -102,6 +102,7 @@
                         AuthenticationService.setLoggedIn(true);
 
                     });
+                    console.log(result);
```

Comment:
Remove all the console.log from code. Only exception is to write a logger.log() method where it is called from all the places.
`var logger = {
log: function(message){
if(isDebugMode){
console.log(message)
}
}
}`